tests/unit/expire.tcl

```diff
@@ -603,9 +603,9 @@ start_server {tags {"expire"}} {
     } {-1}
 
     test {GETEX use of PERSIST option should remove TTL after loadaof} {
+       r config set appendonly yes
        r set foo bar EX 100
        r getex foo PERSIST
-       after 2000
```

Comment:
i don't understand why that sleep was needed. maybe we're missing something?
the expiry was 100 seconds, and we slept just two, so what bug did it attempted to hit?
please dig in the blame log or invite the author to comment.

Reply:
I think it's just copying the code from the test above (the r config set appendonly yes above). we don't need the sleep

Reply:
I was actually debugging another problem and saw this sleep.

the another problem was this (the test below):
```
    test {GETEX propagate as to replica as PERSIST, DEL, or nothing} {
       set repl [attach_to_replication_stream]
       r set foo bar EX 100
       r getex foo PERSIST
       r getex foo
       r getex foo exat [expr [clock seconds]-100]
       assert_replication_stream $repl {
           {select *}
           {set foo bar PXAT *}
           {persist foo}
           {del foo}
        }
        close_replication_stream $repl
    } {} {needs:repl}
```

A daily fails here:   {set foo bar PXAT *}
expected  {del foo15} equal  {set foo bar PXAT *}
i do not recall the specific key name, but was something fooxxx
that is, there is a key expired in the middle
i think it's an unimportant failure, so I didn't go any further
i can find the daily link if needed. (drowned in my daily CI)

Reply:
> adding a FLUSHALL as a first line can solve that.
p.s. we should modify assert_replication_stream to print the whole stream rather than just the failing line.

yes and sure, will make this happen tomorrow